Dangerous property warning

**Describe the bug**
Warning added via: https://github.com/BBC-News/simorgh/pull/921

```
/Users/ashtoc03/Sites/bbc/simorgh/src/server/styles.jsx
  21:26  warning  Dangerous property 'dangerouslySetInnerHTML' found  react/no-danger
```

We should suppress the warning with the `react/no-danger` magic comment.

**To Reproduce**
Steps to reproduce the behavior: `npm test`

**Expected behavior**
There should be no warnings.

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/simorgh/labels/bug)

--- a/src/server/styles.jsx
+++ b/src/server/styles.jsx
@@ -18,6 +18,7 @@ export const getStyleTag = (sheet, isAmp = false) => {
   }, '');
 
   return (
+    // eslint-disable-next-line react/no-danger
     <style amp-custom="" dangerouslySetInnerHTML={{ __html: inlineCss }} />
   );
 };

